scripts/src/errors.ts

```diff
@@ -328,6 +328,6 @@ function DeprecationError(key: keyof typeof CODES.Gone.types, message: string):
 	return new MarketplaceError(CODES.Gone.code, CODES.Gone.types[key], "Request to a deprecated resource", message);
 }
 
-export function BlockchainEndpointChanged(message: string): MarketplaceError {
-	return DeprecationError("BlockchainEndpointChanged", message);
+export function WrongBlockchainVersion(message: string): MarketplaceError {
+	return DeprecationError("WrongBlockchainVersion", message);
```

Comment:
CC @nitzantomer, don't think this affects the client, but the error name changed.

Reply:
Yep, because the old name doesn't fit anymore